index.js

```diff
@@ -1,8 +1,17 @@
+var _ = require('lodash')
```

Comment:
can we just require the functions we really need, like `lodash.frompairs` and `lodash.get`?